src/Tribe/Abstract_Attendance_Totals.php

```diff
@@ -50,4 +50,44 @@ public function integrate_with_attendee_screen() {
 	 * Prints an HTML (unordered) list of attendance totals.
 	 */
 	abstract public function print_totals();
+
+	/**
+	 * Get Attendee Total Sold Tooltip
+	 *
+	 * @since TBD
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_sold_tooltip() {
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
+			<span class="dashicons dashicons-info"></span>
+			<div class="down">
+				<?php echo esc_html_x( 'No matter what the status is, Total Tickets Issued includes how many tickets that have gone through the order process.', 'total sold tooltip', 'event-tickets' ); ?><i></i>
+			</div>
+		</div>
+		<?php
+		return ob_get_clean();
+	}
+
+	/**
+	 * Get Attendee Total Completed Orders Tooltip
+	 *
+	 * @since TBD
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_conpleted_tooltip() {
```

Comment:
Should we move this towards an `admin-view` file?

Reply:
also, typo 🤦‍♂️